packages/ui/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -88,7 +88,8 @@ const itemStyleProps = {
   },
   "data-h2-align-items": "base(center)",
   "data-h2-background-color": "base(transparent) base:hover(secondary.15)",
-  "data-h2-color": "base(secondary.dark) base:selectors[[data-disabled]](gray)",
+  "data-h2-color":
+    "base(black) base:selectors[[data-disabled]](black.lightest)",
```

Comment:
Okay, it looks like `black.light` does work. I also updated the background colour on hover since it is now a little harder to tell what items are disabled.

[fix contrast for disabled items](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7356/commits/85aae5da47bb857f7dcd26d70a04826841e2b325)